### DISCUSSION TITLE
Enzyme localization in pregnant woman model

### DISCUSSION THREAD

**Discussion by KGmax** [Open-Systems-Pharmacology/Forum#1563] [Questions & Problems]
Hi All,
I'm learning to build a pregnant woman model, But I'm a bit confused about the enzyme location in mobi. For CYP2A6 and CYP3A4, even after selecting localization in intracellular in pk-sim, both enzymes show presence in all four subspaces when simulation are built and imported into mobi ?
in pk-sim:

![CYP2A6](https://github.com/Open-Systems-Pharmacology/Forum/assets/107936033/8e2ca0bf-c17e-4edd-9c59-7d51654878aa)

![CYP3A4](https://github.com/Open-Systems-Pharmacology/Forum/assets/107936033/9f974b15-7cf8-4490-a2f5-a54dca4d9118)

in mobi:

![CYP3A4 inter](https://github.com/Open-Systems-Pharmacology/Forum/assets/107936033/951247bc-421e-4db0-a668-0ddde10df3a4)

I am confused by this inconsistency. And do I need to select not is present in interstitial?

**Comment by PavelBal** [comment]
In the model, proteins are located everywhere (so you can easily change the distribution by modifying the "relative expression" parameter). Based on your localization selection in PK-Sim, the parameter `Fraction expressed intracellular` is calculated. You do not have to manually change the `isPresent` property in MoBi.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25903040/507235a1-8561-4b26-a1a5-29274367432d)

**Reply by KGmax** [reply]
Thank you. I will try to understand.

**Comment by Yuri05** [comment]
Please take a look at the documentation, where the localization of proteins is explained in detail:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile
